content/docs/api-reference/dvcfs.md

```diff
@@ -0,0 +1,66 @@
+# DvcFileSystem
+
+DvcFileSystem provides a pythonic file interface (aka
```

Comment:
Maybe it can be further simplified to stress the main points that it's read-only and fsspec comapitable. I think it should start with something like, "DVCFileSystem is a read-only [fsspec](https://filesystem-spec.readthedocs.io/)-compatible interface." Then you could go on to describe the typical operations and everything else.

Reply:
Initially I wrote like that, but I later thought that the end users does not care about fsspec or need to care about it to use it.

Reply:
The `read-only` part is important

Reply:
> Initially I wrote like that, but I later thought that the end users does not care about fsspec or need to care about it to use it.

Good point, but if we are not documenting the available methods, we need to point out clearly that it implements the read-only methods available in fsspec.

Reply:
If you want to first focus on it being a filesytem-like interface and read-only, you could say something like: "DvcFileSystem provides read-only operations like `ls`, `du`, `glob`, `get`, `download`, etc. for a DVC repo. The API implements most read-only methods of [fsspec](https://filesystem-spec.readthedocs.io/)."

Reply:
> the end users does not care about fsspec or need to care about it to use it.

So why do we even mention it?

Reply:
It is fsspec-based filesystem. We do need to mention it, I just don't think we should lead with that in the intro. 🙂

Reply:
Sounds good. Please see https://github.com/iterative/dvc.org/pull/3932#issuecomment-1246342519 below. Thanks